Modules/Journey/QuestDetailsFrame.lua

```diff
@@ -111,7 +133,7 @@ function _QuestieJourney:DrawQuestDetailsFrame(container, quest)
         QuestieJourneyUtils:Spacer(startNPCGroup)
 
         -- Also Starts
-        if startNpc.questStarts then
+        if startNpc.questStarts and #startNpc.questStarts > 1 then
```

Comment:
If the NPC starts any other quests we display them, but hide this section if they don't